apps/cms/lib/repo.ex

```diff
@@ -50,20 +56,19 @@ defmodule CMS.Repo do
 
   @spec news_entry_by(Keyword.t()) :: NewsEntry.t() | :not_found
   def news_entry_by(opts) do
```

Comment:
Is there a reason this page isn't cached?

Reply:
Yeah, this goes to a "view" in Drupal. We don't have a mechanism for the CMS to tell us that a view has been updated. Though, I think it's possible. So, for now we're removing the caching.